src/api/schema/metrics/mod.rs

```diff
@@ -30,6 +31,9 @@ pub use source::{IntoSourceMetrics, SourceMetrics};
 pub use transform::{IntoTransformMetrics, TransformMetrics};
 pub use uptime::Uptime;
 
+#[cfg(feature = "sources-host_metrics")]
+pub use host::HostMetrics;
```

Comment:
Is this used outside of this module?

Reply:
Nope! Nixed this line (along with the `cfg` guard) in favour of `host::HostMetrics` where used 👍🏻